components/brave_rewards/browser/rewards_notification_service_impl.cc

```diff
@@ -351,7 +352,7 @@ void RewardsNotificationServiceImpl::OnFetchPromotions(
     args.push_back(base::NumberToString(item->claimable_until * 1000));
 
     bool only_once = true;
-  #if defined(OS_ANDROID)
+#if BUILDFLAG(IS_ANDROID)
     only_once = false;
   #endif
```

Comment:
Nit: Can we fix up the spacing on the closing #endif as well?

Reply:
oops! I thought `git cl format` would fix that but it didn't. Fixed in https://github.com/brave/brave-core/commit/23457292cae92ce151138c85b8c80552362c527f